### ISSUE TITLE
Review design for responsive 

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/network#58]
We need to look at how the design will work on small screens and super wide ones. We should have comps to clarify items like the hero, nav, join form, and specific content components. I don't think we need complete comps for every page. 

First step: review wireframes and make a list of things to address.

**Comment by xmatthewx** [comment]
- [x] navigation: https://github.com/mozilla/network/issues/72
 - [ ] ??? other concerns?

**Comment by taisdesouzalessa** [comment]
We started this project by setting smaller typefaces sizes without changing the line-height. After doing the home page in mobile version, we noticed that the line height had big gaps, which can be problematic for a site with lots of content like this one. So we tweaked the typeface size and line height (after checking with Gavin that it is possible) to set a comfortable reading for the audience. Here is the before and after the adjustment:

![mobile-mln](https://cloud.githubusercontent.com/assets/22157921/23081737/0e84a052-f50b-11e6-8f18-d81cd500ba4b.png)

Now we fit more elements into the screen. The decisions we made for the home page will affect the other pages so it takes some time to figure it out at the beginning (home page has around 10 typeface styles and 2 patterns that repeats in other pages - top nav and "join our ranks" component). The other pages will be faster after that since we set many styles in the home page already.

**Comment by taisdesouzalessa** [comment]
We also started exploring the "people" page and we are solving the challenge of having a lot of copy + fit the picture and still keep the layout balanced (we found a way to do it and we will update once we have a mockup):

<img width="319" alt="screen shot 2017-02-17 at 12 26 52 pm" src="https://cloud.githubusercontent.com/assets/22157921/23082057/87059c24-f50c-11e6-94d6-af2b4f81bb46.png">

**Comment by taisdesouzalessa** [comment]
Here is the final home page (mobile):
https://invis.io/ZKAIKN1MN#/219686025_MOBILE_-_Home_Page_Copy

@xmatthewx @sabrinang

**Comment by xmatthewx** [comment]
Superb

**Comment by xmatthewx** [comment]
Let's put a note on these comps. A fake post-it not in Sketch artwork. Something like: "Comp for responsive layout. Latest content is in desktop views."

We can update these when big changes happen. But we don't want to maintain two sets for every little copy change.

**Comment by taisdesouzalessa** [comment]
Here is the "About Us" page: https://invis.io/ZKAIKN1MN#/219706000_MOBILE---About-Us

@gvn: the space between paragraphs is too big (see A) if we do a full line break there. Is it feasible to define a custom distance only between paragraphs (see B)? It is just a nice to have not a must have but we were wondering about the effort to do that: easy or difficult? We know it is possible but we don't know if we will be adding too many exceptions and making the code difficult to maintain. We would like to hear your perspective on it.

![question-paragraph](https://cloud.githubusercontent.com/assets/22157921/23086963/5430a1c2-f525-11e6-9288-30c04a136aa4.png)

cc: @sabrinang

**Comment by gvn** [comment]
Yeah, we can control the margin between paragraph elements.

**Comment by taisdesouzalessa** [comment]
News Page: https://invis.io/ZKAIKN1MN#/219715200_MOBILE-News

**Comment by taisdesouzalessa** [comment]
2 more screens ready: People and Programs - I am tying all the pages together in a mobile prototype: https://invis.io/ZKAIKN1MN#/219686025_MOBILE_-_Home_Page_Copy

I recommend you see this URL in a phone screen so you can experience the mobile view. You can tap the menu and it should work. The following links work: People/Programs/About and News (this one can be accessed via home page only).

**Comment by taisdesouzalessa** [comment]
Internet Health Report Interstitial and Upcoming pages are ready. All the responsive pages are in the prototype: https://invis.io/ZKAIKN1MN#/220168109_MOBILE-HomePage - please notice that Campaigns doesn't have a responsive version yet (depends on #34) and Programs too (depends on #38). 

In the mobile prototype you can also:
- interact with the "more details"/"see quote" in the People Page (Hera's card)
- tap the checkbox of the Private Notice (home page) and click on Join Us to see the embedded Thank You message.
@xmatthewx @gvn @sabrinang

**Comment by taisdesouzalessa** [comment]
@gvn 
Sketch files are in the dev folder: _PROJECTS/Mozilla Network Site/dev/all-wireframes-ui-mobile-sketch

The font styles for mobile are in the page called "styles".

**Comment by xmatthewx** [comment]
@gvn – this is good and ready for you to review designs. 
 - Note: We're ignoring the campaign page for now on purpose. You can too. We still need to consider the Projects page. But I wouldn't think that blocks a first pass on implementing this.

@sabrinang @taisdesouzalessa – The one thing I wondered about were the tags on the people page. Should we collapse these into a drop-down? I also think we should remove center alignment on text, except in the offerings. it's works well in those boxes but elsewhere feels out of sync with the rest of the site.

**Comment by sabrinang** [comment]
Hmm I would prefer not to collapse the tag information as a dropdown. It makes the profiles less scanable for how they are related to internet health and it won’t be more than 5 tags. Also, it is a small space and it may feels like UI for options to select that's specific for the profile. If we need to save on vertical space, I think collapsing information should be a global option at the top (like a view mode) and not individual dropdowns.

+1 to left align the thank you message to make it look cohesive with the previous left-aligned form.

**Comment by xmatthewx** [comment]
Fair point on how the tags give context about internet issues. We don't need to save on vertical space. Maybe there's another way to organize the tags so they're more anchored and less floaty.

**Comment by xmatthewx** [comment]
@taisdesouzalessa to make small adjustments commented in invision.

**Comment by taisdesouzalessa** [comment]
To make it less floaty we could add some grey boxes to the background, like that:

<img width="322" alt="screen shot 2017-02-27 at 3 36 49 pm" src="https://cloud.githubusercontent.com/assets/22157921/23385069/92c63a98-fd02-11e6-914e-e0bbacf7c562.png">

However I was thinking about the users' state of mind when they reach this page. I assume they are in a more of exploratory frame of mind (who are these people? What do they do?). So maybe in such a tiny screen like mobile these filters occupy a lot of real estate. What if we did something like this for mobile? https://drive.google.com/a/mozilla.com/file/d/0B3b1KWRwjxtoaHp0eS1vaTZIWFk/view?usp=sharing

Curious to hear your thoughts on it @sabrinang @xmatthewx 

- I will remove center alignment as proposed by you both.

**Comment by xmatthewx** [comment]
Hi @taisdesouzalessa you image didn't seem to upload properly. 

Let's keep our eye on MVP. We can explore more rich ways to explore things post-launch. 

In fact – @gvn shouldn't implement filter tags on People page until after we've landed the page with a solid set of people. Filter tags help frame and communicate things, but they are useless without the profiles below.

**Comment by taisdesouzalessa** [comment]
@xmatthewx good point. Should I remove the tags from the mockup for now then? Or are we still going with the tags for V1 (MVP)?

**Comment by gvn** [comment]
@taisdesouzalessa @xmatthewx Can we close this ticket or are there still things needed from engineering?

**Comment by taisdesouzalessa** [comment]
@gvn we were discussing about having tags on the People's page but I think we can move the discussion to individual tickets instead. This thread is very long.
